docs/serverless-v2/08-04-trigger-lambda-with-event.md

```diff
@@ -58,9 +70,30 @@ Follows these steps:
     EOF
     ```
 
-    The **spec.filter.attributes.eventtypeversion** parameter points to the specific event version, such as `v1`, and **spec.filter.attributes.type** points to the given event type, such as `user.created`.
+    </details>
+    <details>
+    <summary label="console-ui">
+    Console UI
+    </summary>
+
+> **NOTE:** Serverless v2 is an experimental feature, and it is not enabled by default in the Console UI. To use its **Functions [preview]** view, enable **Experimental functionalities** in the **General Settings** view before you follow the steps. Refresh the page after enabling this option.
+
+1. From the drop-down list in the top navigation panel, select the Namespace in which your Application exposes events.
+
+2. Go to the **Functions [preview]** view at the bottom of the left navigation panel and navigate to your lambda.
+
+3. Once in the lambda view, select **Add Event Trigger** in the **Event Triggers** section.
+
+4. Select the event type and version that you want to be a trigger for your lambda and select **Add** to confirm changes.
+
+The `Event Triggers are successfully created` message appears and you will see the trigger available in the **Event Triggers** section in your lambda.
+
+    </details>
+</div>
+
+## Test the trigger
 
-## Trigger the lambda
+> **CAUTION:** Before you follow steps in this section and send an example event, bear in mind that your changes will be propagated to all services subscribed to the event type you send.
```

Comment:
```suggestion
> **CAUTION:** Before you follow steps in this section and send an example event, bear in mind that your events will be propagated to all services subscribed to the event type you send.
```

Reply:
`> **CAUTION:** Before you follow steps in this section and send a sample event, bear in mind that it will be propagated to all services subscribed to this event type.`